REL: 5.0.0 (#2851)
py-pdf/pypdf: CHANGELOG.md

# CHANGELOG

> ## Version 5.0.0, 2024-09-15

Reviewer: Could we please have the introductory text with the highlights here as well?

Author: added

> ### Maintenance (MAINT)

Reviewer 2: ```suggestion
### Deprecations (DEP)
```

> - Deprecate PdfMerger, AnnotationBuilder and other deprecations cleanup (#2813)

Author: They were just deprecating, providing a warning. Now they have been removed as it was announced

> - Deprecate PdfMerger, AnnotationBuilder and other deprecations cleanup (#2813)

Reviewer: We might want to adapt this in the changelog nevertheless, for example using something like "Remove the deprecated PfdMerger and AnnotationBuilder classes and other deprecations cleanup"

> - Deprecate PdfMerger, AnnotationBuilder and other deprecations cleanup (#2813)

Author: The wording is in line with the previous entries in changelog.md. However OK for this rewording

> - Deprecate PdfMerger, AnnotationBuilder and other deprecations cleanup (#2813)

Reviewer 3: I agree with @stefan6419846's rewording as I think it's more correct in its intention which is that the code was removed and it no longer usable. You _have_ to move off PdfReader. For `PdfMerger` specifically, for v3.9.1, there was already:

> * Deprecate PdfMerger (#1866)

So it doesn't make sense to me that it's been doubly deprecated? The use of "Remove" is inline with the v4 wording as well:

> * Remove deprecated code (#2367)
> * Remove deprecated XMP properties (#2386)

# CHANGELOG

## Version 5.0.0, 2024-09-15

> ** Notice : this version drops support to Python 3.7 (not maintained since June 23), PdfMerger(use PdfWriter instead) and AnnotationBuilder(use annotations instead) **

Reviewer: ```suggestion
This version drops support for Python 3.7 (not maintained since July 2023), PdfMerger (use PdfWriter instead) and AnnotationBuilder (use annotations instead).
```

# CHANGELOG

## Version 5.0.0, 2024-09-15

### Security (SEC)
- Fix GitHub workflow vulnerable to script injection (#2787)

### New Features (ENH)
- Add capability to remove /Info from PDF (#2820)
- Robustify parsing for Object streams in XRef rebuilding (#2818)
- Add incremental capability to PdfWriter (#2811)

> - add capability to remove /Info from pypdf

Reviewer 2: ```suggestion
```

That exact string is mentioned a few lines before

- Add UniGB-UTF16 encodings (#2819)
- Robustify on missing font for Tf  operator in text_extract() (#2816)

> -  add incremental capability to PdfWriter

Reviewer 2: ```suggestion
```

That exact string is mentioned a few lines before
